src/submission/models.py

```diff
@@ -228,9 +228,9 @@ def article_media_upload(instance, filename):
 STAGE_CHOICES = [
     (STAGE_UNSUBMITTED, 'Unsubmitted'),
     (STAGE_UNASSIGNED, 'Unassigned'),
-    (STAGE_ASSIGNED, 'Assigned'),
-    (STAGE_UNDER_REVIEW, 'Under Review'),
-    (STAGE_UNDER_REVISION, 'Under Revision'),
+    (STAGE_ASSIGNED, 'Assigned to Editor'),
```

Comment:
A migration will be required to update article.stage from the old to the new values

Reply:
Rightly so, talking utter crap here! Looks fine :+1: